tests/conftest.py

```diff
@@ -46,10 +48,11 @@ class Comment(Base):
 
 
 @pytest.fixture
-def package_path():
+def package_path() -> Generator[Path, None, None]:
     template_path = Path(os.path.dirname(os.path.realpath(__file__))) / "assets"
     with tempfile.TemporaryDirectory() as package_path:
         shutil.copytree(template_path, package_path, dirs_exist_ok=True)
+        os.chdir(package_path)
```

Comment:
Note: This is to have tests closer to their real-life scenarios, because we deal with cwd.